internal/rdma/rdma-test.go

```diff
@@ -224,3 +228,95 @@ func ValidateRDMAResults(results map[string]string) (bool, error) {
 	// If everything is valid
 	return true, nil
 }
+
+// DeleteMofedDeiverNode delete mofed inventory.
+func DeleteMofedDeiverNode(clientset *clients.Settings, podName, namespace, nodeName string) (string, error) {
+
+	pod := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: podName,
+		},
+		Spec: corev1.PodSpec{
+			HostPID:       true,
+			HostNetwork:   true,
+			NodeName:      nodeName,
+			RestartPolicy: corev1.RestartPolicyNever,
+			Volumes: []corev1.Volume{
+				{
+					Name: "host-root",
+					VolumeSource: corev1.VolumeSource{
+						HostPath: &corev1.HostPathVolumeSource{
+							Path: "/",
+						},
+					},
+				},
+			},
+			Containers: []corev1.Container{
+				{
+					Name:    "debugger",
+					Image:   "busybox",
```

Comment:
The busybox container is in Dockerhub I believe and we need an account to pull it.  I suggest using another container image that we can pull in our repo.